docs/runbooks/CnaoDown.md

```diff
@@ -2,29 +2,31 @@
 
 ## Meaning
 
-Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
-This alert fires when CNAO operator is down.
+The `Cluster-network-addons-operator` (CNAO) deploys additional networking components on top of the cluster.
+This alert fires when CNAO is down.
 
 ## Impact
 
-With CNAO operator down, changes in the components are not reconciled, and as a result might not be properly deployed.
+With CNAO down, the cluster cannot reconcile changes to virtual machine components. As a result, the changes might fail to take effect.
 
 ## Diagnosis
 
 - Check CNAO's operator pod namespace:
 	```
-	export NAMESPACE="$(kubectl get deployment -A | grep cluster-network-addons-operator | awk '{print $1}')"
+	$ export NAMESPACE="$(kubectl get deployment -A | grep cluster-network-addons-operator | awk '{print $1}')"
 	```
 
-- Check to see if CNAO's operator pod is down.
+- Check to see if CNAO's operator pod is down:
 	```
-	kubectl -n $NAMESPACE get pods -l name=cluster-network-addons-operator
+	$ kubectl -n $NAMESPACE get pods -l name=cluster-network-addons-operator
 	```
  
-- Check CNAO's operator pod logs and describe.
-    - `kubectl -n $NAMESPACE describe pods -l name=cluster-network-addons-operator`
-    - `kubectl -n $NAMESPACE logs -l name=cluster-network-addons-operator`
+- Check CNAO's operator pod logs and description:
```

Comment:
Same changes as comments above (remove 'operator' and apostrophe). Also, break this into two steps (describe pod, check logs).

Steps need to be numbered. Remember that with markdown, you can use `1.` for every step. This makes updating procedures a lot easier because you do not have to renumber the steps.

Reply:
Ah, TIL :-)